sulfnucrate.F90

```diff
@@ -4,51 +4,386 @@
 
 !!  Calculates particle production rates due to nucleation <rhompe>:
```

Comment:
in sulfnuc.F90, it does homogeneous nucleation first and then uses the rstar value with the heterogeneous nucleation. I don't know if this is still fine when zhao is replaced by vehkamaki or not or whether the rstar needs to be calculated as part of the heterogeneous nucleation path. This is a question for Mike. However, in the stand-alone there needs to be two tests added to exercise this code. They can be copies of the carma_sulfhettest and the carma_sulfatetest, which the nucleation method switched from zhao to vehkamaki.